lab_1_classify_by_unigrams/main.py

```diff
@@ -84,7 +129,25 @@ def compare_profiles(
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
-
+    # shared_tokens = []
```

Comment:
It is necessary to obtain predictions and actual values ​​in order to calculate the MSE. Compare the frequencies of the same symbols in an unknown_profile and in a profile_to_compare. I hope this will help you. Good luck!

Reply:
done